Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -550,11 +575,17 @@ def get_intel_docs(client: Client, data_args: dict) -> Tuple[str, dict, Union[li
     params = assign_params(name=data_args.get('name'), description=data_args.get('description'),
                            type=data_args.get('type'), limit=convert_to_int(data_args.get('limit')),
                            offset=convert_to_int(data_args.get('offset')), labelId=data_args.get('label_id'),
-                           mitreTechniqueId=data_args.get('mitre_technique_id'))
-    raw_response = client.do_request('GET', '/plugin/products/detect3/api/v1/intels/', params=params)
+                           mitreTechniqueId=data_args.get('mitre_technique_id')) if data_args else {}
+    raw_response = client.do_request('GET', '/plugin/products/'
+                                            f'{client.get_threat_response_endpoint()}'
+                                            f'/api/v1/intels/', params=params)
 
     intel_docs = []
     intel_doc = {}
+
+    if client.api_version == '4.x':
+        raw_response = raw_response.get('data')
```

Comment:
Is `data` present in pre-4.x? Can't you use `normalize_api_response()`?

Reply:
no its not present

Reply:
Then please replace the if condition with `normalize_api_response()`